client/my-sites/invites/invite-accept/index.jsx

```diff
@@ -91,6 +88,129 @@ export default React.createClass( {
 		);
 	},
 
+	acceptedNotice( invite ) {
+		invite = invite || this.state.invite;
+		let displayOnNextPage = true;
+		switch ( get( invite, 'role' ) ) {
+			case 'follower':
+				this.props.successNotice(
+					this.translate(
+						'You are now following %(site)s', {
+							args: { site: get( invite, 'site.title' ) }
+						}
+					),
+					{
+						button: this.translate( 'Visit Site' ),
+						href: get( invite, 'site.URL' ),
+						displayOnNextPage
+					}
+				);
+				break;
+			case 'administrator':
+				this.props.successNotice(
+					<div>
```

Comment:
This stuff will be problematic in store dehydration.
If it works now, cool, but generally we only want to put basic types to the store, because we intend it to be serializable